### PR TITLE
Saga correlation id assignments removed

### PR DESCRIPTION
**Pull request by Scooletz**
Removal of all correlation ID assignments which are no longer needed in v6. Solves #94

### CODE REVIEW

**Comment by Scooletz** [comment]
Ping @Particular/azure-storage-persistence-maintainers

File: src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ protected override void ConfigureHowToFindSaga(SagaPropertyMapper<RequestRespons
 
                 public class RequestResponseRequestingSagaData : ContainSagaData
                 {
-                    public virtual Guid CorrIdForResponse { get; set; } //wont be needed in the future
+                    public virtual Guid CorrIdForResponse { get; set; }
```
Comment (SeanFeldman) [inline src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs]: Would expect the property to go, not the comment :)

**Comment by SeanFeldman** [comment]
Nicely timed with the blog post :smiley:

**Comment by Scooletz** [comment]
Yeah. I was considering putting it in the PR comment :) Frankly speaking, it reminded me of this issue.

File: src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs (hunk 1 above)
Comment (Scooletz) [inline src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs]: :)
I read this case thoroughly and I can tell that the correlation id is not set. I can't see why this id should be deleted though. The setter is no longer used, but we need the correlation between request-response? Can you see what's in my blind spot?

Comment (SeanFeldman) [inline src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs]: Looking at the `ConfigureHowToFindSaga` code, `CorrIdForResponse` property on the saga data is used to map the incoming messages. Which means by default, `Data.CorrIdForResponse` is assigned by NSB. Why the comment in place says "won't be needed in the future" now puzzles me as well. This is used to perform the correlation and w/o the endpoint simply won't work (ie test will fail).

Saying that, there's really no test. I can't see any `[Test]` or `Should_`.  What is my blind point?! :confused:

Comment (Scooletz) [inline src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs]: I digged through the history of this tests, including the rename after changing folder structure. It looks like the intial commit of this is here: https://github.com/Particular/NServiceBus.Persistence.AzureStorage/commit/d3342af56b720aa69e9c60bffeb06901a09f0eaf . This is dated Dec 2015. I'd day that we should drop the comment according to our current understanding and leave this property as an existing but no longer assigned correlation id.
What's your opinion?

Comment (Scooletz) [inline src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs]: I'm removing this test. This was a dead test for long and it has nothing to do with the persistence.

**Comment by Scooletz** [comment]
Ping @SeanFeldman ready to be merged

**Comment by SeanFeldman** [comment]
done

**Pull request merged**